armi/mpiActions.py

```diff
@@ -439,6 +440,8 @@ def invokeHook(self):
             context.MPI_SIZE = mpiSize
             context.MPI_NODENAMES = mpiNodeNames
 
+        return actionResult
```

Comment:
I'm not super pumped about adding code and not testing it.

But both of the changes here _are_ in the MPI/parallel space, and I know it's harder to write useful tests here. Hmm